# Speed in the X direction (forward)
float32 speed_x

# Rotational speed around the Z axis
float32 speed_z

# Battery power level
float32 power 

uint8 hand_capture 